server.js

```diff
@@ -4,8 +4,8 @@ const bodyParser = require("body-parser")
 require('dotenv').config()
 const cookieParser = require("cookie-parser")
 
-const username = "Fiamma";
-const password = "IKilledThePope123";
+const { users } = require('./users')
+const { sessions } = require('./users')
```

Comment:
Don't require the same module twice


```suggestion
const { users, sessions } = require('./users')
```